pom.xml

```diff
@@ -4,9 +4,38 @@
     <groupId>com.coveros</groupId>
     <artifactId>selenified</artifactId>
     <version>3.0.1-SNAPSHOT</version>
+
+    <prerequisites>
+        <maven>3.0.4</maven>
+    </prerequisites>
+
     <properties>
+        <java.version>1.8</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
         <suiteXmlFile>integration.xml</suiteXmlFile>
```

Comment:
is there a maven naming convention that we are not following?

Reply:
Not sure which names are are asking about, but mostly no. There are some built-in parameters you weren't using at all (project.build.sourceEncoding) and some that you've created duplicates for (skipITs vs skip.integration.tests).

The lack of defined encodings was creating the "Build is platform-dependent" warnings during builds.